src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1735,6 +1735,7 @@ internal enum ErrorCode
         #endregion diagnostics introduced for C# 8.0
 
         ERR_InternalError = 8751,
+        ERR_VarianceInterfaceNesting = 8752,
```

Comment:
>ERR_VarianceInterfaceNesting [](start = 8, length = 28)

I think this error code should be moved into the section of "diagnostics introduced for C# 8.0"